sklearn/ensemble/tests/test_forest.py

```diff
@@ -1697,11 +1697,16 @@ def test_max_samples_boundary_regressors(name):
     assert ms_1_ms == pytest.approx(ms_None_ms)
 
 
+# TODO: Trees do not preserve dtype when fitting/predicting.
+# Add `global_dtype` when Trees have this ability.
```

Comment:
I think we can just add a sole comment at the top of this file, before the first test:
```python
# TODO: Tree-based model do not preserve dtype on their fitted attribute
# Add a test using `global_dtype` when this is the case.
```